scripts/cbmc_json_parser.py

```diff
@@ -165,6 +171,60 @@ def extract_solver_information(cbmc_response_json_array):
 
     return properties, solver_information
 
+def postprocess_results(properties):
+    """
+    Check for certain cases, e.g. a reachable unsupported construct or a failed
+    unwinding assertion, and update the results of impacted checks accordingly.
+    1. Change all "SUCCESS"/"FAILURE" results to "UNKNOWN" if the reachability
+    check for a Rust construct that is not currently supported by RMC failed,
+    since the missing modeling of the unsupported construct may impact results.
+    2. TODO: Change results from "SUCCESS" to "UNKNOWN" if an unwinding
+    assertion failed, since the insufficient unwinding may cause some execution
+    paths to be left unexplored.
+    Additionally, print a message at the end of the output that indicates if any
+    of the special cases above was hit.
+    """
+
+    has_reachable_unsupported_constructs, has_failed_unwinding_asserts = check_special_cases(properties)
+
+    for property in properties:
+        if has_reachable_unsupported_constructs:
+            # Change status of all properties except failed unsupported
+            # construct reachability checks to UNKNOWN
+            description = property["description"]
+            if GlobalMessages.UNSUPPORTED_CONSTRUCT_DESC not in description or property["status"] == "SUCCESS":
```

Comment:
Can we keep the FAILED properties as FAILED?

Reply:
Yes! Done.